.pylintrc

```diff
@@ -543,7 +543,7 @@ exclude-too-few-public-methods=
 ignored-parents=
 
 # Maximum number of arguments for function / method.
-max-args=5
+max-args=6
```

Comment:
I am fine with increasing this value even more, maybe up to 8?

Then we can remove `# pylint: disable=too-many-arguments` line from several places within our code. I think it makes sense as it's unlikely that we can significantly reduce number of parameters in most of the functions.

Reply:
I prefer a function with less number of arguments. Eight seems to be a lot. IMO its better to use `disable=too-many-arguments` in some cases than explicitly tell that 8 is ok :)

Reply:
I see your point. However, in some cases we're unable to reduce the number of parameters in many functions and honestly, the pylint comments aren't exactly enhancing our code's quality. But, fair enough, let's stick with six for now as you've suggested.